<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QRenderOrangGUI</class>
 <widget class="QMainWindow" name="QRenderOrangGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>568</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>QRenderOrang SVN [dev only release] [*]</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="3,0,1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4,3">
        <item>
         <widget class="QTreeWidget" name="treeShaderStruct">
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tabEditorVert">
           <attribute name="title">
            <string>Vertex Program</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tabEditorFrag">
           <attribute name="title">
            <string>Fragment Program</string>
           </attribute>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frameGLPreview">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="textCompileLog">
        <property name="font">
         <font>
          <family>Bitstream Vera Sans Mono</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>682</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Close"/>
    <addaction name="action_Save"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuShader">
    <property name="title">
     <string>&amp;Shader</string>
    </property>
    <addaction name="action_Compile"/>
    <addaction name="action_LoadSample"/>
   </widget>
   <widget class="QMenu" name="menuA_bout">
    <property name="title">
     <string>A&amp;bout</string>
    </property>
    <addaction name="action_Qt"/>
    <addaction name="action_QRO"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Preview"/>
    <addaction name="action_Messages"/>
    <addaction name="action_Fullscreen"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
   <addaction name="menuShader"/>
   <addaction name="menuA_bout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_Compile">
   <property name="text">
    <string>&amp;Compile</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="action_Qt">
   <property name="text">
    <string>&amp;Qt</string>
   </property>
  </action>
  <action name="action_QRO">
   <property name="text">
    <string>QRender&amp;Orang</string>
   </property>
  </action>
  <action name="action_Preview">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Preview</string>
   </property>
  </action>
  <action name="action_Messages">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Messages</string>
   </property>
  </action>
  <action name="action_Fullscreen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Fullscreen</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="action_LoadSample">
   <property name="text">
    <string>&amp;Load Sample Shader</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
